products/mongodb.md

```diff
@@ -10,18 +10,22 @@ activeSupportColumn: false
 releaseDateColumn: true
 command: mongod --version
 releases:
+  - releaseCycle: "5.0"
+    eol: false
+    release: 2021-07-13
+    latest: "5.0.3"
   - releaseCycle: "4.4"
     eol: false
     release: 2020-07-30
-    latest: "4.4.5"
+    latest: "4.4.9"
   - releaseCycle: "4.2"
     eol: false
     release: 2019-08-31
-    latest: "4.2.13"
+    latest: "4.2.16"
```

Comment:
4.2.17 is already release https://github.com/mongodb/mongo/releases/tag/r4.2.17

Reply:
@BiNZGi thanks so much for the heads up. I got confused with their official page that said it's upcoming 🙈 .

Quick question since this is my first time pushing to this repo: Is it ok to turn the PR into 2 commits or does it need to be squashed into 1?

Reply:
### Do as much commits you need, we squash them at the end :+1: